extension/chrome/elements/compose.ts

```diff
@@ -160,6 +161,7 @@ export class ComposeView extends View {
     this.recipientsModule = new ComposeRecipientsModule(this);
     this.sendBtnModule = new ComposeSendBtnModule(this);
     this.pwdOrPubkeyContainerModule = new ComposePwdOrPubkeyContainerModule(this, storage.hide_message_password);
+    this.fesUrl = storage.fesUrl;
```

Comment:
Overlaps with https://github.com/FlowCrypt/flowcrypt-browser/pull/4231 - one of us will merge it once one PR gets to master.

Reply:
@rrrooommmaaa this looks a bit simpler than #4231 - if we store `this.fesUrl` (which may be undefined) then you don't need the `isFesUsed` methods. So you could remove them all & the code will be a bit more conscise.